src/sentry/models/group.py

```diff
@@ -459,7 +460,7 @@ class Meta:
     __repr__ = sane_repr("project_id")
 
     def __str__(self):
-        return f"({self.times_seen}) {self.error()}"
+        return f"({self.times_seen}) {self.title}"
```

Comment:
If I'm honest, I'm not sure what this change, is.  @mikejihbe ?

Reply:
This removed a deprecation error I ran into. Unrelated to this changeset.